### PR TITLE
Ruby 2.0.0 compatibility

### PR DESCRIPTION
**Pull request by sebastian-nanek**
(Possibly) provide compatibility for ruby 2.0 branch. Let me know if I missed anything.

### CODE REVIEW

File: lib/yu.rb [hunk 1]
```diff
@@ -250,7 +250,10 @@ def template_dir
       relative_path.realpath
     end
 
-    def run_command(command, showing_output: true, exit_on_failure: true)
+    def run_command(command, options={})
```
Comment (mikekelly) [inline lib/yu.rb]: Doesn't ruby 2.0 have named params with default values? I don't think this needs changing

File: lib/yu.rb [hunk 2]
```diff
@@ -220,7 +220,7 @@ def get_existing_services(service_names)
       service_names.select { |service_name| Pathname(service_name).exist? }
     end
 
-    def copy_template_into_dir(service_name:)
+    def copy_template_into_dir(service_name)
```
Comment (mikekelly) [inline lib/yu.rb]: Because this is a named param with no default value, I would make this `options` and do `options.fetch(:service_name)`

Comment (sebastian-nanek) [inline lib/yu.rb]: Changed.

File: lib/yu.rb (hunk 1 above)
Comment (sebastian-nanek) [inline lib/yu.rb]: yes, you are right; I was so overeager building backwards compatibility that I pushed changes which would make it look like 1.9.x :)

**Comment by mikekelly** [comment]
released

**Pull request merged**